lib/node/rpc.js

```diff
@@ -2585,6 +2592,269 @@ class RPC extends RPCBase {
     throw new Error('Fork not found.');
   }
 
+  async decodePSBT(args, help) {
+    if (help || args.length !== 1)
+      throw new RPCError(errs.MISC_ERROR, 'decodepsbt');
+    const valid = new Validator(args);
+    const str = valid.str(0, '');
+    let psbt;
+    try {
+      psbt = PSBT.fromRaw(str, 'base64');
```

Comment:
Why are you expecting base64 format here instead of hex? Is that part of the BIP174 spec?